<launch>
  <arg name="camera_focal_length_x" default="843.3250" />
  <arg name="camera_focal_length_y" default="843.3250" />
  <arg name="image_width" default="752" />
  <arg name="image_height" default="480" />
  <arg name="model_STL_file_path" default="$(find linemod_pose_estimation)/config/stl/coke.stl" />
  <arg name="linemod_template_output_path" default="$(find linemod_pose_estimation)/config/data/coke_linemod_ensenso_templates.yml" />
  <arg name="linemod_renderer_params_output_path" default="$(find linemod_pose_estimation)/config/data/coke_linemod_ensenso_renderer_params.yml" />

  <node name="linemod_renderer_node" pkg="linemod_pose_estimation" type="linemod_renderer_node" output="screen" respawn="false" args="$(arg camera_focal_length_x) 
                      $(arg camera_focal_length_y) 
 		      $(arg image_width)
		      $(arg image_height)
                      $(arg model_STL_file_path) 
                      $(arg linemod_template_output_path) 
                      $(arg linemod_renderer_params_output_path)"
  />

</launch>
